build.rs

```diff
@@ -36,10 +36,17 @@
 // SOFTWARE.
 
 extern crate gcc;
+use std::env;
 
 fn main() {
-    gcc::Build::new()
-               .file("src/traceme.c")
-               .include("src")
-               .compile("traceme");
+    let mut c_build = gcc::Build::new();
+    c_build.file("src/traceme.c").include("src");
+
+    // Travis' Linux setup is too old for what we do with linux-perf.
+    // Setting -DTRAVIS causes the C API to be stubbed.
+    if env::var("TRAVIS").is_ok() {
```

Comment:
Should (can?) we check *which* version of Travis? If/when they upgrade, maybe we should have an `assert` (or similar) to alert us of this, and make us investigate building things fully?

Reply:
We can check the output of `lsb_release -r` if we are on travis and panic if the version changed.

Reply:
Sounds like a plan!